client/data/support-article-alternates/use-support-article-alternates-query.js

```diff
@@ -0,0 +1,29 @@
+import { isDefaultLocale } from '@automattic/i18n-utils';
+import { useQuery } from 'react-query';
+import { useSelector } from 'react-redux';
+import wp from 'calypso/lib/wp';
+import { isPostKeyLike } from 'calypso/reader/post-key';
```

Comment:
The dependency on `reader` here is a bit unfortunate, and can be replaced by a simple `!!(blogId && postId)` check.

Reply:
Not the best solution, but simple enough to do the job. Done in 0e5ca18419a64845a63db67e95c7df051e322e7f